locales/en/index.yml

```diff
@@ -2901,11 +2901,24 @@ support:
       contactSupport: "Contact support"
       seeReports: "See reports in progress"
   askPermissions:
-    header: "Contact support"
-    title: "To continue, we need your consent"
+    nameSurname: "Name and Surname"
+    fiscalCode: "Fiscal code"
+    emailAddress: "Email address"
+    deviceAndOS: "Device and Operating System"
+    ipAddress: "IP address"
+    currentAppVersion: "Version of the app in use"
+    identityProvider: "Identity Provider"
+    diagnosticData: "In-app navigation data"
```

Comment:
@debiff this is now called "In-app navigation data" (e.g.: the last screen you visited), so the string "diagnosticData" should be changed (See Figma for more details)